tools/codegen/dest/lazy_ir.py

```diff
@@ -241,18 +255,32 @@ def this_shape(i: int) -> str:
                 shapes_str = ','.join([this_shape(i) for i in range(returns_length)])
                 meta_out = "std::vector<Shape> shapes{" + shapes_str + "};"
 
-            meta_str = f"""auto out_meta = at::meta::{schema.aten_name}({', '.join(str(a.name) for a in all_args)});
-        {meta_out}"""
+            meta_str = f"""
+            auto out_meta = at::meta::{schema.aten_name}({', '.join(str(a.name) for a in all_args)});
+            {meta_out}"""
         else:
             shape_sig = ComputeShapeSignature(metadata.kernel, func)
             meta_str = f"""
-        auto shapes = {shape_sig.shape_call};"""
+            auto shapes = {shape_sig.shape_call};"""
 
         meta_str += f"""
-        TORCH_INTERNAL_ASSERT(shapes.size() == {returns_length});"""
+            TORCH_INTERNAL_ASSERT(shapes.size() == {returns_length});"""
+
+        symbole_str = aten_symbol(schema)
+        # Rewrite inplace symbol like relu_ into relu
+        # TODO: zero_ is an exception. Need figure out why.
+        if symbole_str == "at::aten::relu_":
+            symbole_str = symbole_str[:-1]
```

Comment:
hmm. do we actually want to truncate the symbol? let's assume that for some reason we actually want to support `relu_` instead of relying on the copy trick above.  then why not use its actual _ symbol?